README.md

```diff
@@ -99,13 +97,8 @@ the end of this file).
   1. Install the prerequisites.
   2. Clone the Hercules repository (see [GitHub](https://github.com/HerculesWS/Hercules)) using a git client, into a new
      folder.
-  3. Connect to the MySQL server as root:
-    - Create a database (hercules): `CREATE DATABASE hercules;`
-    - Create a user (hercules): `CREATE USER 'hercules'@'localhost' IDENTIFIED BY 'password';`.
-    - Give permissions (GRANT SELECT,INSERT,UPDATE,DELETE) to the user: `GRANT SELECT,INSERT,UPDATE,DELETE ON hercules.* TO 'hercules'@'localhost';`
-  4. Connect to the MySQL server as the new user:
-    - Import the .sql files in /sql-files/ into the new database.
-  5. Start Visual Studio and load the provided solution:
+  3. Run mariadb.bat to automatically install and configure MariaDB.
+  4. Start Visual Studio and load the provided solution:
```

Comment:
We could even reduce this further to a single command
```batch
powershell -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://rawgit.com/HerculesWS/Hercules/stable/install.ps1'))"
```

We could make said command do the following:
- Install scoop,
- Install git
- Install MariaDB
- Install Visual Studio
- Clone Hercules
- Configure MariaDB
- Compile Hercules

Reply:
power shell probably better leave as optional command
because not all windows may have and support powershell.

Reply:
powershell is present since XP SP2, and XP and Vista aren't even supported anymore and 7 is only still supported in business editions

we can safely rely on powershell

Reply:
i not have poweshell in xp sp3, but probably it can be installed.
not sure is other windows is powershell optional component or not.

Reply:
Windows 10 have powershell.
XP didn't have it.
Powershell was added to windows from Windows 7

Reply:
There you go, I added a powershell detection in `mariadb.bat`